packages/esm-patient-chart-app/src/ui-components/patient-chart.component.tsx

```diff
@@ -19,6 +22,7 @@ const PatientChart: React.FC<RouteComponentProps<PatientChartParams>> = ({ match
   const { patientUuid, view, subview } = match.params;
   const [loading, patient] = useCurrentPatient(patientUuid);
   const state = useMemo(() => ({ patient, patientUuid }), [patient, patientUuid]);
+  const { active } = useWorkspace();
```

Comment:
I am not sure if we need all this.

CSS should be sufficient. There are plenty of selectors in CSS (e.g., `:empty`, adjacent children, ...) which should make it possible to support the desktop view without any functional changes.

The problem when introducing functional changes just to achieve layout is that this won't scale. And it introduces spaghetti code. I'd appreciate if we either stick to CSS only, or properly encapsulate the changes (effectively also not requiring changes to the existing code). The existing code should be free of any "desktop" or "mobile" considerations. Exception: The real set of functionalities is different between desktop and mobile.